docs/src/modules/components/BaseUIComponents.js

```diff
@@ -169,7 +169,12 @@ function BaseUIComponents() {
                 }),
               })}
             />
-            <Typography component="h2" variant="body2" fontWeight={600} sx={{ px: 2, py: 1.5 }}>
+            <Typography
+              component="h2"
+              variant="body2"
+              fontWeight="semiBold"
+              sx={{ px: 2, py: 1.5 }}
+            >
```

Comment:
Replace the hard-coded 600 value for the design token `theme.typography.fontWeightSemiBold`